variants/vmware-k8s-1.21/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+# This is the vmware-k8s-1.21 variant.  "." is not allowed in crate names, but
+# we don't use this crate name anywhere.
+name = "vmware-k8s-1_21"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant]
+image-format = "vmdk"
+supported-arches = ["x86_64"]
+kernel-parameters = [
```

Comment:
Looks like you started before the kdump support was merged, we'll want to add that.